symphony/bdk/core/service/message_service.py

```diff
@@ -0,0 +1,366 @@
+        if kwargs.get("attachment") is not None:
+            attachment = kwargs["attachment"]
+            kwargs["attachment"] = attachment if type(attachment) is list else [attachment]
+        if kwargs.get("preview") is not None:
+            preview = kwargs["preview"]
+            kwargs["preview"] = preview if type(preview) is list else [preview]
+
+        params = {
+            'sids': stream_ids,
+            'session_token': self._auth_session.session_token,
+            'key_manager_token': self._auth_session.key_manager_token,
+            'message': message
+        }
+        params.update(kwargs)
+        return await self._messages_api.v4_multi_attachment_message_blast_post(**params)
+
+    async def import_messages(self, messages: [V4ImportedMessage]) -> [V4ImportResponse]:
+        params = {
+            'message_list': V4MessageImportList(value=messages),
+            'session_token': self._auth_session.session_token,
+            'key_manager_token': self._auth_session.key_manager_token
+        }
+        import_response_list = await self._messages_api.v4_message_import_post(**params)
+        return import_response_list.value
+
+    async def get_attachment(self, stream_id: str, message_id: str, attachment_id: str):
+        params = {
+            'sid': stream_id,
+            'file_id': attachment_id,
+            'message_id': message_id,
+            'session_token': self._auth_session.session_token,
+            'key_manager_token': self._auth_session.key_manager_token
+        }
+        return await self._attachment_api.v1_stream_sid_attachment_get(**params)
+
+    async def suppress_message(self, message_id: str) -> MessageSuppressionResponse:
+        params = {
+            'id': message_id,
+            'session_token': self._auth_session.session_token
+        }
+        return await self._message_suppression_api.v1_admin_messagesuppression_id_suppress_post(**params)
+
+    async def get_message_status(self, message_id: str) -> MessageStatus:
+        params = {
+            'mid': message_id,
+            'session_token': self._auth_session.session_token
+        }
+        return await self._message_api.v1_message_mid_status_get(**params)
+
+    async def get_attachment_types(self) -> [str]:
+        params = {
+            'session_token': self._auth_session.session_token
+        }
+        type_list = await self._pod_api.v1_files_allowed_types_get(**params)
+        return type_list.value
+
+    async def get_message(self, message_id: str) -> V4Message:
+        params = {
+            'id': message_id,
+            'session_token': self._auth_session.session_token,
+            'key_manager_token': self._auth_session.key_manager_token
+        }
+        return await self._messages_api.v1_message_id_get(**params)
+
+    async def list_attachments(self, stream_id: str, **kwargs) -> [StreamAttachmentItem]:
```

Comment:
is the kwargs temporary? if we use it don't we loose completion/documentation?

Reply:
I don't really understand your point that the kwargs temporary. IMHO, the **kwargs contains all the optional fields that we gonna pass to the api and we have to document all the fields that we can pass to the api through kwargs

Reply:
and what about using default arguments? (named ones like here https://www.geeksforgeeks.org/default-arguments-in-python/)
could that help to be more explicit?